Manual/Tutorial.tex

```diff
@@ -110,10 +110,12 @@ \subsection{Implementation}
 objects of the same category.
 
 After the declarations of the constructors we create a global category \GAP object, which will be the category $\Svec$.
-Since we are going to create an Abelian category, we tell the system that we are doing this, to have all the computational
-properties of an Abelian category. 
-Next, we tell \CapPkg that every \GAP object which lies in the filter \texttt{IsSQVecObj} is an object of $\Svec$,
+The call of \texttt{CreateCapCategoryWithDataTypes} lets \CapPkg create a category with name
+``\texttt{Skeletal category of rational vector spaces}'' such that every \GAP object which lies in the filter \texttt{IsSQVecObj} is an object of $\Svec$,
 and every \GAP object which lies in the filter \texttt{IsSQVecMor} is a morphism of $\Svec$.
```

Comment:
Technically this is wrong, the converse is correct: Every object of Svec lies in `IsSQVecObj`, but an object in `IsSQVecObj` does not necessarily belong to Svec. But this was there before, so we can defer this to another PR, your choice.

Reply:
I corrected it.